com.microsoft.mrtk.input/Simulation/Prefabs/MRTKInputSimulator.prefab

```diff
@@ -68,6 +68,10 @@ MonoBehaviour:
     simulationMode: 1
     anchorPoint: 0
     changeNeutralPose: {fileID: -1599196918749775551, guid: 70b361cf3c037d54896482d1ffd9f999, type: 3}
+    defaultGesture: 2
```

Comment:
Do we also want to align "neutral" vs "default"? or are those conceptually different?

Reply:
I'll change it to default! Neutral is too suggestive of a "resting" pose in my opinion.

To add clarity. I don't want to change the semantics we have in place for the synthetic hands at this time, in part because it changes between the "neutral" and "select" pose based off of the value from the select input action

Reply:
> Neutral is too suggestive of a "resting" pose in my opinion.

I'm fine with this change, but I do think that was the point 🙂 the difference between the "resting" pose and the "action" pose.

Reply:
I think it'd be better to reconsider any naming considerations in a future PR, since the refactoring considerations are getting a bit entangled with initial goal of adding some the simulator improvements.

Reply:
If that's the case, I'd prefer to undo any Pose/Gesture renames in this PR and follow-up with that discussion. It doesn't make sense to me to deprecate methods with one terminology when we're postponing a terminology decision to after this PR.

![image](https://user-images.githubusercontent.com/3580640/184459889-bb31bb7f-3bb1-4cb3-ae80-0f4ed325716a.png)

Reply:
Changed Default to Neutral with this feedback!